content/next/index.md

```diff
@@ -28,10 +30,10 @@ The commit contains the following structural elements, to communicate intent to
 consumers of your library:
 
 1. **fix:** a commit of the _type_ `fix` patches a bug in your codebase (this correlates with [`PATCH`](http://semver.org/#summary) in semantic versioning).
-2. **feat:** a commit of the _type_ `feat` introduces a new feature to the codebase (this correlates with [`MINOR`](http://semver.org/#summary) in semantic versioning).
-3. **BREAKING CHANGE:** a commit that has the text `BREAKING CHANGE:` at the beginning of its optional body or footer section introduces a breaking API change (correlating with [`MAJOR`](http://semver.org/#summary) in semantic versioning).
+1. **feat:** a commit of the _type_ `feat` introduces a new feature to the codebase (this correlates with [`MINOR`](http://semver.org/#summary) in semantic versioning).
```

Comment:
Is there a reason that all of these have been changed to `1.` rather than a numbered list, this seems like a bug if it's required for hugo to render them.

Reply:
It's not required by HUGO.
It will automatically number them in the generated HTML, the main advantage is when editing the list adding elements in the middle the diff will be shorter because you won't have to change manually the numbers.

Reply:
that makes sense to me, less work to edit 👍